Cannot find module './.env' or its corresponding type declarations

<!--
PLEASE HELP US PROCESS GITHUB ISSUES FASTER BY PROVIDING THE FOLLOWING INFORMATION.

ISSUES MISSING IMPORTANT INFORMATION MAY BE CLOSED WITHOUT INVESTIGATION.
-->

## I'm submitting a...
<!-- Check one of the following options with "x" -->
- [x] Bug report  <!-- Please search GitHub for a similar issue or PR before submitting -->
- [ ] Feature request
- [ ] Documentation issue or request
- [ ] Question

## Current behavior

On very first `npm start` right after creating a new project, I get the following error:
```
ERROR in src/environments/environment.ts:11:21 - error TS2307: Cannot find module './.env' or its corresponding type declarations.

11 import { env } from './.env';
```

## Expected behavior
no error message


## Minimal reproduction of the problem with instructions
1. `npm install -g generator-ngx-rocket`
2. `ngx new`
3. `npm start`

## Environment

<pre><code>

ngX-Rocket: 9.1.0
Node.js: v15.0.1
Npm: 7.0.3
OS: win32 x64 10.0.19041

Generated project options:
{
  "generator-ngx-rocket": {
    "version": "9.1.0",
    "props": {
      "location": "path",
      "strict": false,
      "skipInstall": false,
      "skipQuickstart": false,
      "initGit": true,
      "usePrefix": true,
      "appName": "rocket2",
      "target": [
        "web"
      ],
      "pwa": false,
      "ui": "ionic",
      "layout": "side-menu",
      "auth": false,
      "lazy": false,
      "angulartics": false,
      "languages": [
        "de-DE",
        "en-US"
      ],
      "tools": [
        "prettier",
        "hads"
      ],
      "utility": [
        "lodash"
      ],
      "deploy": "none",
      "projectName": "rocket-2",
      "packageManager": "npm",
      "mobile": [],
      "desktop": []
    }
  }
}

     _                      _                 ____ _     ___
    / \   _ __   __ _ _   _| | __ _ _ __     / ___| |   |_ _|
   / △ \ | '_ \ / _` | | | | |/ _` | '__|   | |   | |    | |
  / ___ \| | | | (_| | |_| | | (_| | |      | |___| |___ | |
 /_/   \_\_| |_|\__, |\__,_|_|\__,_|_|       \____|_____|___|
                |___/


Angular CLI: 10.1.7
Node: 15.0.1
OS: win32 x64

Angular: 10.2.2
... animations, common, compiler, compiler-cli, core, forms
... language-service, platform-browser, platform-browser-dynamic
... router
Ivy Workspace: Yes

Package                         Version
---------------------------------------------------------
@angular-devkit/architect       0.1001.7
@angular-devkit/build-angular   0.1001.7
@angular-devkit/core            10.1.7
@angular-devkit/schematics      10.1.7
@angular/cli                    10.1.7
@schematics/angular             10.1.7
@schematics/update              0.1001.7
rxjs                            6.6.3
typescript                      4.0.5

</code></pre>

--- a/generators/app/templates/_package.json
+++ b/generators/app/templates/_package.json
@@ -8,18 +8,18 @@
   <% } -%>
   "scripts": {
     "ng": "ng",
-    "build": "<%= run('env') %> && ng build --prod",
-    "start": "<%= run('env') %> && ng serve --proxy-config proxy.conf.js",
+    "build": "<%= run('write:env') %> && ng build --prod",
+    "start": "<%= run('write:env') %> && ng serve --proxy-config proxy.conf.js",
 <% if (props.deploy !== 'none') { -%>
-    "deploy": "<%= run('env') %> && ng deploy",
+    "deploy": "<%= run('write:env') %> && ng deploy",
 <% } -%>
 <% if (props.pwa) { -%>
     "serve:sw": "<%= run('build') %> && npx http-server ./<%= props.target.includes('cordova') ? 'www': 'dist' %> -p 4200",
 <% } -%>
     "lint": "ng lint && stylelint \"src/**/*.scss\" --syntax scss && htmlhint \"src\" --config .htmlhintrc",
-    "test": "<%= run('env') %> && ng test",
-    "test:ci": "<%= run('env') %> && <%= run('lint') %> && ng test --configuration=ci",
-    "e2e": "<%= run('env') %> && ng e2e",
+    "test": "<%= run('write:env') %> && ng test",
+    "test:ci": "<%= run('write:env') %> && <%= run('lint') %> && ng test --configuration=ci",
+    "e2e": "<%= run('write:env') %> && ng e2e",
     "translations:extract": "ngx-translate-extract --input ./src --output ./src/translations/template.json --format=json --clean --sort",
 <% if (props.tools.includes('hads')) { -%>
     "docs": "hads ./docs -o",
@@ -27,7 +27,7 @@
 <% if (props.tools.includes('compodoc')) { -%>
     "compodoc": "compodoc -p tsconfig.json -d docs/generated -s -o",
 <% } -%>
-    "env": "ngx-scripts env npm_package_version",
+    "write:env": "ngx-scripts env npm_package_version",
 <% if (props.tools.includes('prettier')) { -%>
     "prettier": "prettier --write \"./{src,e2e}/**/*.{ts,js,html,scss}\"",
     "prettier:check": "prettier --list-different \"./{src,e2e}/**/*.{ts,js,html,scss}\"",
@@ -43,7 +43,7 @@
 <% if (props.target.includes('electron')) { -%>
     "electron:compile": "tsc --project tsconfig.electron.json",
     "electron:run": "<%= run('electron:compile') %> && electron .",
-    "electron:build": "<%= run('electron:compile') %> && <%= run('env') %> && ng build --prod --base-href ./dist.electron --output-path dist.electron",
+    "electron:build": "<%= run('electron:compile') %> && <%= run('write:env') %> && ng build --prod --base-href ./dist.electron --output-path dist.electron",
     "electron:package:common": "electron-packager . --overwrite --out=dist.packages --ignore=\"/(platforms|plugins|docs|www|reports|e2e|src)\"",
 <%   if (props.desktop.includes('windows')) { -%>
     "electron:package:windows": "<%= run('electron:package:common') %> -- --platform=win32 --arch=x64 --icon=./resources/win/icon.ico",

